playground/.vuepress/styles/palette.styl

```diff
@@ -0,0 +1,8 @@
+
+$msAccentDarkColor = $accentColor
+
+$msInputDarkBgColor = #444d52
+
+$msTextDarkColor = #eaeaea
+
+$borderDarkColor = lighten($msInputDarkBgColor, 10%)
```

Comment:
Can you add a new line at the end of the line? We want to keep a consistent style to avoid this kind of update in a future PR that is not related at all to this file 🙂

To avoid this kind of error, you can download the editorconfig plugin, depending on your text editor. After downloading it, every time you will save your file, it will automatically apply the basic style rules to your file (removing trailing whitespaces, adding a new line at the end of file...) thanks to the .editorconfig file in this repo.
See more about editorconfig: https://editorconfig.org/#overview
And the editorconfig file of this repo: https://github.com/meilisearch/meilisearch-ruby/blob/master/.editorconfig

Reply:
Done ! Indeed I didn't had the plugin in vscode. Thanks for the explanation ! ❤️